### PR TITLE
Project class and list endpoint

### PR DESCRIPTION
**Pull request by CarolinWickner**
- Created a project class
- Created endpoint to list all projects (hard-coded for now)

### CODE REVIEW

File: src/AppBundle/Controller/ProjectController.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace AppBundle\Controller;
+
+header('Content-Type: application/json');
```
Comment (mfranczak) [inline src/AppBundle/Controller/ProjectController.php]: Good idea!
However please take a look on symfony 3.4 documentation regarding JsonResponse.
https://symfony.com/doc/3.4/components/http_foundation.html#creating-a-json-response
```
The JsonResponse class sets the Content-Type header to application/json and encodes your data to JSON when needed.
```

File: src/AppBundle/Controller/ProjectController.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+use Symfony\Component\HttpFoundation\Response;
+use AppBundle\Entity\Project;
+
+class ProjectController extends Controller
+{
+   /**
+    * @Route("/projects")
+    * @Method("GET")
+    */
+
```
Comment (mfranczak) [inline src/AppBundle/Controller/ProjectController.php]: When you write comments to methods there should be no newline between method signature and ending of your multiline comment.
```
/**
 * my multiline comment
 */
 private function hello() {
   //...
 }
```

Comment (CarolinWickner) [inline src/AppBundle/Controller/ProjectController.php]: I've found an easier way to serialise: https://stackoverflow.com/questions/7005860/php-json-encode-class-private-members

File: src/AppBundle/Controller/ProjectController.php (hunk 1 above)
Comment (CarolinWickner) [reply src/AppBundle/Controller/ProjectController.php]: So I guess we don't need this line to set content header.
Btw, I've found a nicer way to serialise: https://stackoverflow.com/questions/7005860/php-json-encode-class-private-members

**Pull request merged**